javascript/index.js

```diff
@@ -14,15 +14,24 @@ let milUni = document.getElementById('milUni');
 let splits = document.getElementById('splits');
 
 function printTime() {
-  // ... your code goes here
+  getMinutes()
+  getSeconds()
```

Comment:
getMinutes and getSeconds are methods of the chronometer, the functions that you wanted to call here are printMinutes and printSeconds, that are running the above methods.